contrib/mako/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "mako"
+pkgver = "1.8.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dbash-completions=true",
+    "-Dfish-completions=true",
+    "-Dzsh-completions=true",
+]
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "scdoc",
+]
+makedepends = [
+    "cairo-devel",
+    "elogind-devel",
+    "gdk-pixbuf-devel",
+    "linux-headers",
+    "pango-devel",
+    "wayland-devel",
+    "wayland-protocols",
+]
+depends = [
+    "dbus",
```

Comment:
there is no need to depend on dbus, nor it is a good idea (presumably the user will have a bus through other means, and it does not necessarily mean that it's *this* dbus)

Reply:
since basu does not split the progs, you get it regardless, due to having a library dependency